tree 30793dd931ed
parent 2a6cc4c73e22
author Benjamin Zores <ben@geexbox.org> 1264374749 -3600
committer Benjamin Zores <ben@geexbox.org> 1264374749 -3600
revision 2815
branch enna

fix enna crash when using fb engine
diff --git a/src/bin/enna.c b/src/bin/enna.c
--- a/src/bin/enna.c
+++ b/src/bin/enna.c
@@ -109,7 +109,8 @@
 #ifdef BUILD_ECORE_X
 static void _mouse_display(int show)
 {
-    ecore_x_window_cursor_show(enna->ee_winid, show);
+    if (strcmp (enna_config->engine, "fb"))
+        ecore_x_window_cursor_show(enna->ee_winid, show);
     enna->cursor_is_shown = show;
 }
 
